webpack/assets/javascripts/react_app/components/ResourceErrors/ResourceErrorsActions.js

```diff
@@ -0,0 +1,16 @@
+import {
+  RESOURCE_ERRORS_RESOLVE,
+  RESOURCE_ERRORS_RERUN,
+} from './ResourceErrorsConstants';
+
+export const resolveResourceErrors = resourceErrors => dispatch =>
```

Comment:
These actions aren't asynchronous, so we can just create an action creator ( a function that returns an object):
```js
const doAddToDoItem = payload => ({ type: 'TODO_ADDED', payload })
```
 For async actions we can use `thunk` and then the action returns another function with `dispatch` as an argument:
```js
export const incrementAsync = () => dispatch =>  {
    setTimeout(() => {
      dispatch(increment());
    }, 1000);
  };
}
```